import.py

```diff
@@ -339,7 +339,9 @@ def _report_environment():
     repo_privacy = True if os.environ.get('CF_IMPORT_PRIVATE_REPO', 'true').strip().lower() == "true" else False  # noqa: E501
     gitea_url = f'{gitea_base_url.rstrip("/")}/api/v1'
     org = os.environ.get('CF_IMPORT_GITEA_ORG', 'cray').strip()
-    repo_name = os.environ.get('CF_IMPORT_GITEA_REPO', product_name + '-config-management').strip()  # noqa: E501
+    repo_name = os.environ.get('CF_IMPORT_GITEA_REPO').strip()  # noqa: E501
+    if not repo_name:
+        repo_name = product_name + '-config-management'
```

Comment:
Need to preserve behavior when this env var is not set. E.g.:

```
>>> os.environ.get('CF_GITEA_IMPORT_REPO').strip()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'strip'
```

```suggestion
    repo_name = os.environ.get('CF_IMPORT_GITEA_REPO', '').strip()  # noqa: E501
    if not repo_name:
        repo_name = product_name + '-config-management'
```